pkgs/top-level/python-packages.nix

```
@@ -5990,6 +5990,8 @@ in {
 
   pygame = callPackage ../development/python-modules/pygame { };
 
+  pygame_gui = callPackage ../development/python-modules/pygame_gui { };
```

```suggestion
  pygame-gui = callPackage ../development/python-modules/pygame-gui { };
```
new packages should be normalized and old ones eventually converted.

---

Apologies. I have reverted it